libs/entry-components/search-filter/search-filter-input/search-filter-input.component.html

```diff
@@ -0,0 +1,27 @@
+<mat-form-field [formGroup]="form">
+  <mat-label [attr.for]="searchFilter.key">{{searchFilter.label}}</mat-label>
+
+  <div [ngSwitch]="searchFilter.controlType">
+
+    <input *ngSwitchCase="controlType.text"
+      [formControlName]="searchFilter.key" [id]="searchFilter.key"
+      [type]="searchFilter.type" matInput [placeholder]="searchFilter.placeholder" [maxlength]="searchFilter.maxLength">
+
+    <mat-select *ngSwitchCase="controlType.select"
+      [formControlName]="searchFilter.key" [id]="searchFilter.key" [multiple]="searchFilter.multiSelect">
+      <mat-option *ngIf="!searchFilter.multiSelect" [value]="undefined">
+        {{config.noneSelectedOptionText}}
+      </mat-option>
+      <mat-option *ngFor="let option of searchFilter.options" [value]="option.key">{{option.label}}</mat-option>
+    </mat-select>
+
+    <mat-select *ngSwitchCase="controlType.dynamicSelect"
```

Comment:
we could use entry-autocomplete komponent here but then we need to add reference to entry-form?

Reply:
If we add this ref then we could re-use SelectOption which will answer my other question

Reply:
we agreed not to re-use autocomplete from entry-form lib